engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-action-packages.controller.js

```diff
@@ -84,5 +84,22 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkActionPackag
             return deferred.promise;
         };
 
+        $scope.performViaRemoteExecution = function(action, customize) {
+            var remoteAction, selectedHosts = $scope.nutupane.getAllSelectedResults();
+            if ($scope.content.contentType == "package_group") {
+                remoteAction = "group_" + action;
+            } else if ($scope.content.contentType == "package") {
+                remoteAction = "package_" + action;
+            }
+            form = $('#packageActionForm');
+            form.attr('action', '/katello/remote_execution');
+            form.attr('method', 'post');
```

Comment:
There is an API in remote execution, that can already be used to achieve this. In fact, I've tried to go the API way first, but failed miserably: my problem was I needed to get to the remote execution UI (either the job invocaiton show page, or the new job invocation form in case of customize first case): I really want to avoid doing the same remote execution UI twice with and without angular. When using the API approach, I then had to use `$window.location.href` for getting to other page + and just translate the API reponse to the `path` to pass to `$window.location.href`. So I explored this path, but turned out to be too complex for what we want to achiieve. Long story short, I ended up going this way because:

1. the main goal for this is getting to the foreman remote execution UI, and I have not found any nice Angular way to do the stuff.
2. there is already API and CLI on the remote execution that can be just used for doing the same thing. Therefore, the added value of having the API on Katello side as well didn't look that interesting.